### ISSUE TITLE
Can't create new rust project

### ISSUE THREAD

**Issue by agracey** [deislabs/spiderlightning#225]
**Description of the bug**
When I try to create a new project, I get this error:

```
agracey@desktop:~/Projects/scratchpad> slight new -n slisp@v0.1 rust
Error: failed to iterate over archive

Caused by:
    failed to fill whole buffer
```

**To Reproduce**
Install slight using https://deislabs.github.io/spiderlightning/#how-to-install-on-macos-and-linux and run the command above. I saw the same output on both my M1 mac and x86 linux computers. 

**Additional context**
Both have cargo/rust installed using rustup. 
```
agracey@desktop:~/Projects/scratchpad> rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/agracey/.rustup

stable-x86_64-unknown-linux-gnu (default)
rustc 1.64.0 (a55dd71d5 2022-09-19)
```

I tried in both fish and bash.

**Comment by Mossaka** [comment]
Thanks for filing the issue! I can reproduce that `slight new` isn't working with the new release with the same error output. It is probably the new release introduced regression on the template tar files. We will look into this issue!

**Comment by Mossaka** [comment]
I just fixed the README to refer to the `0.1.0` version
